### ISSUE TITLE
Include ESLint default config files in app/addon blueprints

### ISSUE THREAD

**Issue by Turbo87** [ember-cli/ember-cli#6864]
Instead of running the `ember-cli-eslint` blueprint on `ember new` we should instead just duplicate the empty ESLint config files so that e.g. they show up at https://github.com/ember-cli/ember-new-output and also work with the `--skip-npm` flag.

**Comment by kellyselden** [comment]
We discussed this before, but remind me. Why can't we just run the generator as part of the release process? It would show up in the npm dist, and we don't have to commit to master and maintain duplicates.

**Comment by Turbo87** [comment]
the things that is most annoying for me personally is https://github.com/ember-cli/ember-cli/blob/master/RELEASE.md#test not working anymore with this setup

**Comment by Turbo87** [comment]
tl;dr is that I think KISS might make sense here instead of making it infinitely complex just to save us from duplicating two files with 18 lines in total...

**Comment by rwjblue** [comment]
Agreed
